### ISSUE TITLE
"Posts.xml" (Line 1081546): JAXP00010004: The accumulated size of entities is "50 000 001" that exceeded the "50 000 000" limit set by "FEATURE_SECURE_PROCESSING".

### ISSUE THREAD

**Issue by flochman** [BaseXdb/basex#1510]
Hi,

I try to create new database in BaseX by importing XML file but got this error:

"Posts.xml" (Line 1081546): JAXP00010004: The accumulated size of entities is "50 000 001" that exceeded the "50 000 000" limit set by "FEATURE_SECURE_PROCESSING".

Posts.xml is over 50GB huge.

How to solve this please?

Thanks.

**Comment by ChristianGruen** [comment]
Sounds like a parsing problem (this error message is not generated by BaseX). Did you try to set INTPARSE to true?

**Comment by flochman** [comment]
Where I can set this option please ?

**Comment by ChristianGruen** [comment]
Please check out our documentation (http://docs.basex.org/) or write to our mailing list for more help.

**Comment by flochman** [comment]
I tried to setup INTPARSE to true in both .basex and .basexgui but the error remain.

Have you any other suggestions?

Thanks.

**Comment by ChristianGruen** [comment]
So you assigned INTPARSE as local option in .basex (i.e., at the end of the config file)? Assigning it in .basexgui won’t work. You can verify it was correctly assigned by calling the command GET INTPARSE, or you can simply enable »Use internal XML parser« in the Parsing tab of the Create Database dialog in the GUI.

> Have you any other suggestions?

Sure, see my previous answer (the mailing list is your friend).

**Comment by flochman** [comment]
Many thanks Christian.

I enabled  »Use internal XML parser« in the Parsing tab of the Create Database dialog in the GUI and error dissapear and now is going proccessing of creation a database.

Previously I edited .basex file and added INPARSE option but whe I looked after running BaseX GUI it was rewrited again without this option.

Many thanks.

I will see how the database will processed. It takes a time to process 50GB XML file.
